### ISSUE TITLE
Merge of resources from app + package

### ISSUE THREAD

**Issue by Levivb** [antonioribeiro/health#159]
Hey,

This package support the possibility to customize the resources in `config/health/resources`. Upon doing so, the resources in the packages are skipped completely. It's quite a shame to copy-paste all resources just because one resource needs a bit of customization.

Would it be possible to merge the resources in the config directory and the resources in the package? That way, most code is maintained in the package and only that what needs to be changed is available in developer projects

Let me know what you think :)

**Comment by kost999** [comment]
It is very useful!
http://joxi.ru/12MgyOqflRVnMr
And can be realised right here by merging files arrays

**Comment by antonioribeiro** [comment]
@Levivb I completely agree with you, when I first built this, I had the impression people would have to customize most of the resources.

Can you provide a Pull Request to do this merge?

**Comment by antonioribeiro** [comment]
I have a question: if we don't copy all those files from the package dir, how will copy and related to them when they need to modify something in the resource?
